deploy/manifests/dev/us-east-2/tenant/storetheindex/instances/ago/deployment.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: indexer
+spec:
+  template:
+    spec:
```

Comment:
Under spec you'll need to patch `volumes` and `volumenMounts` to include the new ad data volume and mount it on the pod. Look at the base overlay deployment as an example.